lib/minitest/test.rb

```diff
@@ -18,6 +18,12 @@ class << self; attr_accessor :io_lock; end # :nodoc:
     self.io_lock = Mutex.new
 
     ##
+    # The signal to use for dumping information to STDERR. Defaults to "INFO".
+
+    class << self; attr_accessor :info_signal; end
+    self.info_signal = "INFO"
+
```

Comment:
Drunk at ruby on ales. I'll get back to you when sober.

> On Mar 31, 2016, at 16:13, Nathan Sutton <notifications@github.com> wrote:
> 
> In lib/minitest/test.rb:
> 
> > @@ -18,6 +18,12 @@ class << self; attr_accessor :io_lock; end # :nodoc:
> >      self.io_lock = Mutex.new
> >  
> >      ##
> > +    # The signal to use for dumping information to STDERR. Defaults to "INFO".
> > +
> > +    class << self; attr_accessor :info_signal; end
> > +    self.info_signal = "INFO"
> > +
> I wasn't sure where to put it and was hoping for feedback. Where would you prefer?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>